build.py

```diff
@@ -1768,6 +1785,29 @@ def backend_build(
     cmake_script.blankln()
 
 
+def backend_clone(
+    be,
+    cmake_script,
+    tag,
+    install_dir,
+    github_organization,
+):
+    repo_clone_dir = os.path.join(install_dir, "backends")
+
+    cmake_script.commentln(8)
+    cmake_script.comment(f"'{be}' backend")
+    cmake_script.comment("Delete this section to remove backend from build")
+    cmake_script.comment()
+    cmake_script.mkdir(repo_clone_dir)
+    cmake_script.cwd(repo_clone_dir)
+    cmake_script.gitclone(backend_repo(be), tag, be, github_organization)
```

Comment:
Additionally, it will be helpful if we can provide vllm version from this script to the backend to use from the VERSION_MAP above.

Reply:
This can be done later.